### DISCUSSION TITLE
[Feature] Multi-endpoint app configuration

### DISCUSSION THREAD

**Discussion by aechmtwash** [immich-app/immich#2701] [Feature Request]
### The feature

Considering this is a self hosted system, it is highly likely that there will be instances where you may want to browse a friends or family members database of photos.  It would be ideal if you could setup the app differently for multiple endpoints.  Say for instance you have your own endpoint, your parents have another, and your in-laws have another.  

At minimum the app should be able to view photos across the different endpoints, but ideally you would also be able to uniquely configure the backups, so only certain albums push to certain endpoints.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by imjuzcy** [comment]
Agreed. It could be like the Nextcloud app, where users can log into different instances, switch between instances and have different settings for different instances.
